### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by apurva1112**

### CODE REVIEW

**Comment by masayyad** [comment]
testing comment

**Review by masayyad — APPROVED** [review]

**Review by masayyad — CHANGES_REQUESTED** [review]
chnages requested test

**Review by ori2111 — APPROVED** [review]

**Comment by ori2111** [comment]
### Installation
[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github). After you've signed in to your Slack workspace, you will be prompted to give the app access:

<p align="center">auth</p>

After the app is installed, and once you've added the GitHub integration to the relevant channels using `/invite @github`, you will see previews of links to GitHub issues, pull-requests, and code rendered as rich text in your workspace.

<p align="center">unfurl_convo</p>

**Comment by ori2111** [comment]
[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github).

**Comment by ori2111** [comment]
### Installation
[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github). After you've signed in to your Slack workspace, you will be prompted to give the app access:

**Comment by ori2111** [comment]
### Installation
[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github).

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ffffaaa
```
Comment (ori2111) [inline README.md:1]: # Here I am starting a thread

**Just testing**

**Comment by ori2111** [comment]
@apurva1112  ssup

**Comment by ori2111** [comment]
[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github). After you've signed in to your Slack workspace, you will be prompted to give the app access:

**Comment by ori2111** [comment]
### Installation
[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github). After you've signed in to your Slack workspace, you will be prompted to give the app access:

**Comment by ori2111** [comment]
### Installation

[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github). After you've signed in to your Slack workspace, you will be prompted to give the app access:

**Comment by ori2111** [comment]
### Installation

[Install the GitHub integration for Slack](https://slack.com/apps/A8GBNUWU8-github)

**Pull request merged**